address.json

```diff
@@ -2,51 +2,29 @@
   "airdropcampaign-polkadot.network": [
     "14FscqFT8S8W8emC5294cEpDctgAucJW7C99mpxS4cucpHoA"
   ],
-  "blockfipro.com": [
-    "15Z7MdMM71vHTL7jNmNhm5xCUG5szJuyccGr3p5uzZgkz6Lv"
-  ],
-  "claimpolka.com": [
-    "12jo5dBMedbma3XcU6Grj7m6ckeupjg4aj9Q1oFyK3Z7oh3"
-  ],
-  "claimpolka.live": [
-    "12aCLUsyTCWn5Dzf5wxuuRm93dx9rkAF5rqw9zBDTV5NSY1E"
-  ],
+  "blockfipro.com": ["15Z7MdMM71vHTL7jNmNhm5xCUG5szJuyccGr3p5uzZgkz6Lv"],
```

Comment:
Please revert the formatting changes here. It is done for the sake of consistency between all, despite the number of entries.

Reply:
shoots sorry, it's my prettier taking control. I'll revert it.